test/Java10andUp/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,11 +1,100 @@
 eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.annotation.inheritNullAnnotations=disabled
```

Comment:
Can you remove this change from this PR?  Updating the eclipse jdt settings should be done in a separate pull request and should be done for all the JavaXandUp directories at the same time.

Reply:
Done.